windows/code/RNDeviceInfoCPP.h

```diff
@@ -58,6 +83,27 @@ namespace winrt::RNDeviceInfoCPP
         return false;
       }
     }
+	
+    REACT_SYNC_METHOD(getDeviceTypeSync);
+    std::string getDeviceTypeSync() noexcept
+    {
+      if (isTabletHelper()) {
+        return "Tablet";
+      }
+      else if (winrt::Windows::System::Profile::AnalyticsInfo::VersionInfo().DeviceFamily() == L"Windows.Xbox")
+      {
+        return "GamingConsole";
```

Comment:
I'm not sure if it's possible now or ever will be, but it would be cool to know what the app was if this code actually shipped in a context where getDeviceType === "GamingConsole" :-) :video_game: